### PR TITLE
[3785] extends PR https://github.com/scandipwa/scandipwa/pull/4093

### PR DESCRIPTION
**Pull request by pablobentivengo**
**Related issue(s):**
* Fixes https://github.com/scandipwa/scandipwa/issues/3785

**Problem:**
* Not scrolling to and shaking unselected options 

**In this PR:**
* Extends PR https://github.com/scandipwa/scandipwa/pull/4093

### CODE REVIEW

**Comment by Elena-Evsa** [comment]
Issue 7: Animation of shaking absent 
https://vimeo.com/669396048/bbaba8bde9

**Comment by aleksandrakorolova** [comment]
1. Animation of shaking is missing for dropdown - https://drive.google.com/file/d/1VOHeyqFWpMcJ_5NBIdNmHWPtZuDT7T0d/view?usp=sharing
2. Please, do animation of shaking more smooth as it was before (https://5-0-0-rc.d25kd40decjpue.amplifyapp.com/default/n-cocktail-dress.html)

**Comment by aleksandrakorolova** [comment]
1. Fixed
2. Fixed
3. Page scrolls to the review/status. User can't see options at all in mobile view if product has long 'short description'
https://drive.google.com/file/d/1PRKSis0VyKsbtku-lboKtexC3cKauHR3/view?usp=sharing
1st part of video - PR
2nd part of video - 5-0-0.
Please, do the same as in 5-0-0

**Comment by aleksandrakorolova** [comment]
3. Fixed
4. Quantity control buttons are shifted up 
Screenshot: lest side - PR, right side - master
Screenshot 2022-02-09 at 07 57 23

**Review by ejnshtein — CHANGES_REQUESTED** [review]

File: packages/scandipwa/src/component/Product/Product.container.js [hunk 1]
```diff
@@ -327,18 +341,51 @@ export class ProductContainer extends PureComponent {
...
     }
 
+    validateConfigurableProduct() {
+        const {
+            parameters
+        } = this.state;
+
+        const { product: { configurable_options } } = this.props;
+        const unselectedOptions = [];
+        Object.keys(configurable_options).forEach((option) => {
```
[the remark this answers is not in the document]
Comment (pablobentivengo) [reply packages/scandipwa/src/component/Product/Product.container.js]: Hey! I ran a test to see performance and it was pretty much the same. Even though I changed it to **reduce**, I think it makes the code shorter and simpler. Let me know what you think.

image

**Review by ejnshtein — APPROVED** [review]

**Pull request merged**